tests/code_style.py

```diff
@@ -13,12 +13,6 @@
 from rdt.transformers.base import BaseTransformer
 
 
-def validate_transformer_name(transformer):
```

Comment:
Since we are renaming the transformer to `LabelEncoder`, I am deleting the quality test which assured all transformers end with the `Transformer` word.